CHANGELOG.md

```diff
@@ -1,14 +1,16 @@
-## 2.2.0-beta.2
+## 2.2.0
  - feat: Improves Serverpod startup and lifecycle events logging.
- - fix: Responds with 400 when throwing serializable exceptions.
- - fix: Enables logging for tests and ensures sessions are always closed.
-
-## 2.2.0-beta.1
  - feat: Adds full support for testing framework.
  - feat: Adds configuration for controlling log output location.
  - feat: Adds support for signing out a user from a single device.
  - feat: EXPERIMENTAL. Adds support for inheritance in models.
  - feat: EXPERIMENTAL. Adds support for sealed classes in models.
+ - fix: Only reports invalid Dart endpoint definition files when using `--watch`.
+ - fix: Uses direct model import when protocol files are analyzed.
+ - fix: Removes redundant file collection from code generation.
+ - fix: Fixes error in `serverpod generate` when a `Session` is set as a named required parameter.
+ - fix: Responds with 400 when throwing serializable exceptions.
+ - fix: Enables logging for tests and ensures sessions are always closed.
```

Comment:
Suggestion: Remove this line, this is a fix for the testing framework that is release with the update.